test/boost/mutation_test.cc

```diff
@@ -108,6 +108,7 @@ with_column_family(schema_ptr s, replica::column_family::config cfg, sstables::s
         auto cm = make_lw_shared<compaction_manager>(tm, compaction_manager::for_testing_tag{});
         auto cl_stats = make_lw_shared<cell_locker_stats>();
         auto s_opts = make_lw_shared<replica::storage_options>();
+        s_opts->value = replica::storage_options::local{ .dir = fs::path(cfg.datadir) };
```

Comment:
Don't we now have a helper for this now (introduced 2 patches ago)?

Reply:
Yes, good point. Likely I introduced the helper after this patch and forgot to update this place when shuffled the branch